views/edit-program.ejs

```diff
@@ -0,0 +1,66 @@
+<%- include('partials/header') %>
+<div class="container">
+<% if (add || program) { %>
+	<form method="post" action="/<%=add ? 'program_add' : `program_edit/${program.id}`%>">
+		<div class="form-row">
+			<div class="form-group col-md-6">
+				<label for="title">Title</label>
+				<input type="text" class="form-control" name="title" value="<%=program.title%>" required>
+			</div>
+			<div class="form-group col-md-6">
+				<label for="description">Description</label>
+				<input type="text" class="form-control" name="description" value="<%=program.description%>" required>
+			</div>
+		</div>
+		<div class="form-row">
+			<div class="form-group col-md-6">
+				<label for="start_date">Start Date</label>
+				<input type="text" class="form-control" name="start_date" value="<%=program.start_date%>" required>
```

Comment:
Try `type="date"` for Start and End date. This should define a date picker and makes sure the format is always MM/DD/YYYY.

Reply:
Complete